packages/core/src/odata-v2/request-builder/create-request-builder.ts

```diff
@@ -34,71 +26,14 @@ export class CreateRequestBuilderV2<EntityT extends EntityV2>
     readonly _entityConstructor: Constructable<EntityT>,
     readonly _entity: EntityT
   ) {
-    super(new ODataCreateRequestConfig(_entityConstructor, oDataUriV2));
-    this.requestConfig.payload = serializeEntityV2(
-      this._entity,
-      this._entityConstructor
-    );
-  }
-
-  get entity(): EntityT {
-    return this._entity;
-  }
-
-  /**
-   * @deprecated Since v1.29.0. This method should never be called, it has severe side effects.   * Builds the payload of the query.
-   * @returns the builder itself
-   */
-  prepare(): this {
```

Comment:
This method is removed, which is fine since it was deprecated.
Please add one line to the compatibility notes.

Reply:
Hmm, I think we should just keep it as it was public API. We should only remove it in version 2.

Reply:
yes, my mistake. I put it back in.